ext/featurefmt/featurefmttest/featurefmttest.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2018 clair authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package featurefmttest contains utility functions for featurefmt tests
+package featurefmttest
```

Comment:
this is to make the testing codebase more like the idiomatic testing code: https://golang.org/src/net/http/httptest/

Reply:
I don't feel too strongly about it, but the featurefmt package itself doesn't contain much and is intended as package that is imported to implement featurefmts. I think it keeps things simple by including everything you need to implement a featurefmt in that package.

Most of the code is around the complexity of reading in test files, which I'd prefer to avoid to keep things simple.